Include stdarg.h for the va_list type exposed by the winpr_vasprintf.

Index: winpr/include/winpr/string.h
--- winpr/include/winpr/string.h.orig
+++ winpr/include/winpr/string.h
@@ -21,6 +21,7 @@
 #ifndef WINPR_CRT_STRING_H
 #define WINPR_CRT_STRING_H
 
+#include <stdarg.h>
 #include <wchar.h>
 #include <stdio.h>
 #include <string.h>
